### ISSUE TITLE
Enable Username and Password Sign-In Option

### ISSUE THREAD

**Issue by kunal2812** [ContriHUB/Youdemy#15]
Currently, our platform only allows users to sign in using their email addresses and passwords. To improve user convenience and flexibility, we aim to introduce the option for users to sign in with their usernames in addition to email addresses. This feature will enhance the UX by accommodating users who prefer usernames for authentication.

Note:

- Create a new branch for every task to isolate changes
- Start working on the issue only when it is assigned to you
- Submit the PR on time, no extra time will be allotted
- Follow the existing theme of website
- Don't request for the issue if it has locked label on it (will be unlocked after dependencies are fixed)

**Comment by Manan-Arora31** [comment]
Can this issue be assigned to me .. ? It shows assignment request cannot be made by you .

**Comment by kunal2812** [comment]
No. of assigned issues + No. of pending requests must be greater than 3 that's why you're not able to request. Complete your pending issues or get your requests cleared to request for new issues.

**Comment by Manan-Arora31** [comment]
I have got my requests cleared .. still then not able to send the request

**Comment by kunal2812** [comment]
Assigned to @Manan-Arora31

**Comment by Shineri** [comment]
Can this issue be assigned to me please?

**Comment by kunal2812** [comment]
ContriHub is now over.
